advanced/sr_gui_cyberglove_calibrator/src/sr_gui_cyberglove_calibrator/cyberglove_calibrator.py

```diff
@@ -366,29 +366,28 @@ def calibrate_current_step(self):
             self.btn_save.setEnabled(True)
 
     def save_calib(self):
-        filename = QtGui.QFileDialog.getSaveFileName(
+        filename = QtWidgets.QFileDialog.getSaveFileName(
             self._widget, 'Save Calibration', '')
         if filename == "":
             return
 
         self.calibrer.write_calibration_file(filename)
 
         # QMessageBox returns 0 for yes
-        if QtGui.QMessageBox.information(self._widget,
-                                         "Load new Calibration",
-                                         "Do you want to load the new calibration file?",
-                                         "yes",
-                                         "no") == 0:
+        if QtWidgets.QMessageBox.information(self._widget,
```

Comment:
I couldn't find anything that specifies that there should be any other difference in this. Your solution might be applicable here but I did not have this error